src/xdocs/config_misc.xml

```diff
@@ -1084,6 +1084,23 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           times it is practical experience. In any case, this check should
           just ensure that a minimal set of indentation rules is followed.
         </p>
+        <p>
+          Basic offset indentation is used for indentation inside code blocks.
```

Comment:
> inside code blocks

Who is used outside code blocks?
like fields or such, or is that considered a code block?

Reply:
I presume any code in `{  }` will follow basic offset.

Reply:
Then what is brace adjustment?

Reply:
from property doc:
> Specify how far a braces should be indented when on the next line.

it is same as all other properties it actually mean `Specify how far line that started with braces should be indented` 
If you like think wording, we can use it for all properties to be a bit more exact.

Reply:
Done.